koiic/vehicle-store-api
# #5 An endpoint to delete a single car object

**What does this PR do?**

An endpoint to delete a single car object

**Description of Task to be completed?**

As a user, I should be able to delete a single car endpoint

**How should this be manually tested?**

- Fetch the branch with `git fetch origin ft-delete-single-user-05 `
- Check out the branch with `git checkout  ft-delete-single-user-05 `
- Run the app: `docker-compose up` or `python manage.py runserver`
- On postman create a get request to this url `http://127.0.0.1:8000/api/car/delete/<int:id>/`

**Any background context you want to provide?**

N/A

**What are the relevant pivotal tracker stories?**

N/A

**Screenshots (if appropriate)**

N/A
